### PR TITLE
Fix indentation bugs and a couple of other issues

### PR DESCRIPTION
**Pull request by Melkiades**
#903 #909 and some from #738

### CODE REVIEW

File: R/tt_export.R [hunk 1]
```diff
@@ -942,7 +964,11 @@ tt_to_flextable <- function(tt,
   flx <- flextable::width(flx, width = final_cwidths) # xxx to fix
 
   if (!is.null(theme)) {
-    flx <- theme(flx)
+    flx <- theme(
+      flx,
+      tbl_ncol_body = NCOL(tt),
+      tbl_row_class = make_row_df(tt)$node_class # These are ignored if not in the theme
+    )
```
Comment (Melkiades) [inline R/tt_export.R:971]: this is why theme should always have ...

Comment (shajoezhu) [reply R/tt_export.R:971]: what if we introduce a default, and dummy theme

Comment (Melkiades) [reply R/tt_export.R:971]: you mean instead of null?

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ## rtables 0.6.9.9007
 ### Features
  * Modified `reorder_split_levels()` to cover more edge cases and be more stringent in the allowed inputs.
+ * Removed `tt` input from `theme_docx_default()` and added internal handling for row classes and number of columns.
+ * Reworked padding and spacinf in default theme `theme_docx_default()`.
```
Comment (Melkiades) [inline NEWS.md:5]:
```suggestion
 * Reworked padding and spacing in default theme `theme_docx_default()`.
```

File: NEWS.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
...
+ * Refactored page size handling for `export_as_docx()`.
```
Comment (Melkiades) [inline NEWS.md:6]:
```suggestion
 * Added top left information handling (now bold and bottom aligned)
```

**Comment by shajoezhu** [comment]
shall we try flextable >= 0.9.6 and officer >= 0.6.6, rspm

**Comment by Melkiades** [comment]
> shall we try flextable >= 0.9.6 and officer >= 0.6.6, rspm

I think it may not be too relevant as the only fix that got in that was necessary comes from https://github.com/davidgohel/flextable/issues/652 which would be release 0.6.8 (when it will arrive). Besides, I do not think the different versions are relevant for us so far

**Review by shajoezhu — APPROVED** [review]
lgtm! thanks @Melkiades

**Pull request merged**